src/common/NoResult/styled.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import { ReactComponent as PlaneImage } from "./../../assets/planeNoResult.svg";
+
+export const HeaderText = styled.header`
+  margin: 56px 0 24px;
+  font-size: 36px;
+  font-weight: 600;
+  line-height: 1.2;
+  color: ${({ theme }) => theme.colors.text};
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px) {
+    font-size: 24px;
+    margin: 42px 0 18px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
+    margin: 24px 0 12px;
+    margin: 16px;
+    font-size: 18px;
+  }
+`;
+
+export const ImageWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 24px;
+`;
```

Comment:
What do you think, maybe it will be look better if image will be at the middle of the page? Just a suggestion.

Reply:
I was thinking about for a while but I decided to keep because figma file doesn't shows very where how to place it, but will be good to care about it at the end when you will discus it together - there is a text and I think probably will be good to push it down a little bit.